### DISCUSSION TITLE
BlockGrid Editor + Macro

### DISCUSSION THREAD

**Discussion by huwred** [umbraco/Umbraco-CMS#13155] [Features and ideas]
Hi,

Firstly I'm loving the new Blockgrid editor, however I was wondering if it is possible to render a Macro in a block? Will this be added or should we be doing it a different way? For example I have a latest posts macro I would like to insert into the blockgrid

**Comment by huwred** [comment]
In answer to my own question, the solution was very simple without needing a macro :)
I just moved the code from my macro partial view into a blockgrid custom view then just created a new element with the same name as the custom view, added any datatypes to that if the macro had parameters, job done.

**Reply by nielslyngsoe** [reply]
Thanks for the feedback and for sharing insight into your solution.
So just to be 100% sure, did this obliterate your need for a Macro completely?

**Reply by huwred** [reply]
yes it did, I was able to recreate them as block items, but seems a bit mute now since it was pulled from Umbraco 10 :(

**Reply by nul800sebastiaan** [reply]
Block Grid will come back to v10 (10.4.0) and is currently also available in the v11 RC (https://umbraco.com/blog/umbraco-11-release-candidate/), sorry for the few weeks of extra delay!
